### ISSUE TITLE
ga4gh/quality-control-wgs metrics_definitions.md - Genome coverage uniformity

### ISSUE THREAD

**Issue by justinjj24** [c-BIG/NPM-sample-qc#87]
>Description: The median absolute deviation of sequencing coverage derived from short paired-end sequencing [high quality](https://github.com/ga4gh/quality-control-wgs/blob/0290612d0237078dea7ae38dc13e680189fc46d1/metrics_definitions/metrics_definitions.md#high-quality-reads), [non duplicated](https://github.com/ga4gh/quality-control-wgs/blob/0290612d0237078dea7ae38dc13e680189fc46d1/metrics_definitions/metrics_definitions.md#duplicated-reads) reads, [primary alignments](https://github.com/ga4gh/quality-control-wgs/blob/0290612d0237078dea7ae38dc13e680189fc46d1/metrics_definitions/metrics_definitions.md#primary-alignments), achieving a [mapping quality](https://github.com/ga4gh/quality-control-wgs/blob/0290612d0237078dea7ae38dc13e680189fc46d1/metrics_definitions/metrics_definitions.md#mapping-quality) of 20 or greater, in [autosomes non gap regions](https://github.com/ga4gh/quality-control-wgs/blob/0290612d0237078dea7ae38dc13e680189fc46d1/metrics_definitions/metrics_definitions.md#autosomes-non-gap-regions) of [GRCh38 assembly](https://github.com/ga4gh/quality-control-wgs/blob/0290612d0237078dea7ae38dc13e680189fc46d1/metrics_definitions/metrics_definitions.md#grch38-assembly). [Clipped bases](https://github.com/ga4gh/quality-control-wgs/blob/0290612d0237078dea7ae38dc13e680189fc46d1/metrics_definitions/metrics_definitions.md#clipped-bases) are expluded. [Overlapping bases](https://github.com/ga4gh/quality-control-wgs/blob/0290612d0237078dea7ae38dc13e680189fc46d1/metrics_definitions/metrics_definitions.md#overlapping-bases) are counted only once. It is critcal that the (BAM/CRAM) alignment files be readily marked for [duplicated reads](https://github.com/ga4gh/quality-control-wgs/blob/0290612d0237078dea7ae38dc13e680189fc46d1/metrics_definitions/metrics_definitions.md#duplicated-reads) and [clipped bases](https://github.com/ga4gh/quality-control-wgs/blob/0290612d0237078dea7ae38dc13e680189fc46d1/metrics_definitions/metrics_definitions.md#clipped-bases).

>Implementation details: In the NPM-sample-QC reference implementation, the genome-wide sequencing coverage of non duplicated reads, non clipped bases, non overlapping bases, primary alignments, achieving a mapping quality of 20 or greater is derived from [mosdepth v0.3.2](https://github.com/ga4gh/quality-control-wgs/blob/0290612d0237078dea7ae38dc13e680189fc46d1/metrics_definitions/metrics_definitions.md#mosdepth). It is further narrowed down to the non gap regions of GRCh38 assembly, autosomes only using [bedtools intersect](https://github.com/ga4gh/quality-control-wgs/blob/0290612d0237078dea7ae38dc13e680189fc46d1/metrics_definitions/metrics_definitions.md#bedtools-intersect). The median absolute deviation of the coverage is then calculated using [datamash](https://github.com/ga4gh/quality-control-wgs/blob/0290612d0237078dea7ae38dc13e680189fc46d1/metrics_definitions/metrics_definitions.md#datamash).

**expluded -> excluded**

Good to include the comment given below or remove the `high quality` from the description? The `high quality` reads also not part of the description in the proposed GHIF WGS QC document!

- NPM-sample-qc implementation metric `Genome coverage uniformity` doesn't select [high quality](https://github.com/ga4gh/quality-control-wgs/blob/0290612d0237078dea7ae38dc13e680189fc46d1/metrics_definitions/metrics_definitions.md#high-quality-reads) reads using `mosdepth`
- **It is critcal that the (BAM/CRAM) alignment files generated only from the illumina PF reads.**

**Comment by mhebrard** [comment]
Fix typo
Keep `high quality`
